src/Chapter1.hs

```diff
@@ -520,8 +521,10 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
-
+closestToZero x y 
+  | abs x <= abs y = x
+  | abs x >= abs y = y
+  | otherwise = error""
```

Comment:
You can use `if-then-else` instead of guards. This will also remove the last redundant `otherwise` case 🧹